src/core_codemods/sql_parameterization.py

```diff
@@ -428,3 +504,18 @@ def leave_Call(self, original_node: cst.Call) -> None:
                                 expr.value
                             ):
                                 self.calls[original_node] = query_visitor.leaves
+
+
+def _extract_prefix_raw_value(self, node) -> Optional[Tuple[str, str]]:
```

Comment:
kinda similar to `clean_simplestring` which is in another util module. At some point we should try to combine